e2e/test/scenarios/question/notebook.cy.spec.js

```diff
@@ -444,8 +444,11 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
     });
 
     cy.findByTestId("fields-picker").click();
-    cy.findAllByLabelText("More info").first().realHover();
-    cy.findByRole("dialog").contains("This is a unique ID");
+    popover().within(() => {
+      cy.findAllByLabelText("More info").first().realHover();
+    });
```

Comment:
We usually skip `within()` if there's going to be only one expression in the callback function.
This comment applies to the other modified e2e tests as well.

```suggestion
    popover().findAllByLabelText("More info").first().realHover();
```